docs/pages/guides/configuring-ota-updates.md

```diff
@@ -10,11 +10,11 @@ OTA updates are controlled by the [`updates` settings in app.json](../workflow/c
 
-With this automatic configuration, calling [`Updates.reloadAsync()`](../versions/latest/sdk/updates.md#updatesreloadasync) will also result in Expo attempting to fetch the most up-to-date version of your app, so there is no need to use any of the other methods in the Updates module.
+The timeout length is configurable by setting `updates.fallbackToCacheTimeout` (ms) in app.json. For example, a common pattern is to set `updates.fallbackToCacheTimeout` to `0`. All new apps with `expo-cli` have this set in `app.json`. This will allow your app to start immediately with a cached bundle while downloading a newer one in the background for future use. When using this configuration, users that download the app from a store and launch it for the first time will always see the version of the app that the binary was built against. [`Updates.addListener`](../versions/latest/sdk/updates.md#expoupdatesaddlistenereventlistener) provides a hook to let you respond when the new bundle is finished downloading. You can use this to notify users that they should restart the app, and you can also restart it programmatically with [`Updates.reloadAsync()`](../versions/latest/sdk/updates.md#updatesreloadasync), eg: in response to a prompt to the user.
 
-The timeout length is configurable by setting `updates.fallbackToCacheTimeout` (ms) in app.json. For example, a common pattern is to set `updates.fallbackToCacheTimeout` to `0`. This will allow your app to start immediately with a cached bundle while downloading a newer one in the background for future use. [`Updates.addListener`](../versions/latest/sdk/updates.md#expoupdatesaddlistenereventlistener) provides a hook to let you respond when the new bundle is finished downloading.
+With this automatic configuration, calling [`Updates.reloadAsync()`](../versions/latest/sdk/updates.md#updatesreloadasync) will also result in Expo attempting to fetch the most up-to-date version of your app, so there is no need to use any of the other methods in the Updates module.
```

Comment:
This is actually no longer true -- this was the behavior of the old `Updates.reload` method (when `Updates.reloadFromCache` also existed), but it no longer applies, so we should just get rid of this whole sentence.